### ISSUE TITLE
why add a interleave before the input?

### ISSUE THREAD

**Issue by williamium3000** [kekmodel/FixMatch-pytorch#50]
Wonderful job! 
I want to know what role does do_interleave function do, is the performance dropping when removing it?

    logits = de_interleave(logits, 2*args.mu+1)

**Comment by cjc20000323** [comment]
> Wonderful job! I want to know what role does do_interleave function do, is the performance dropping when removing it?
> 
> ```
> logits = de_interleave(logits, 2*args.mu+1)
> ```

Do you know how the interleave performs? I don't think it is useful.

**Comment by williamium3000** [comment]
It's not useful. I think it's just inherented from the original tf implementation where it matters.

**Comment by cjc20000323** [comment]
您好，我已收到您的邮件，请您知悉。
